config/secrets.py.example

```diff
@@ -24,6 +24,14 @@ elif LOCALHOST:
     UPLOADS_DEFAULT_DEST = BASE_DIR + '/uploads/'
     UPLOADS_DEFAULT_URL = 'http://localhost:5000/uploads/'
 
+
+# Database backup path. PRODUCTION and STAGING can be ignored for
```

Comment:
There is no more PRODUCTION and STAGING as it relates to the backup dest, so this comment needs to be updated.